_rules/role-required-states-and-properties-4e8ab6.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: 4e8ab6
-name: Role has required states and properties
+name: Element with role has required states and properties
```

Comment:
explicit should also be in the name

Reply:
Agreed, though wondering if explicit or implicit

Reply:
Explicit ... ie. elements with a role attribute. eg. `<div role="main">